src/mode/modeHandler.ts

```diff
@@ -1277,6 +1376,33 @@ export class ModeHandler implements vscode.Disposable {
         }
       }
 
+      // Check if the selection we are going to set is different than the current one.
+      // If they are the same vscode won't trigger a selectionChangeEvent so we don't
+      // have to add it to the ignore selections.
+      const willTriggerChange =
+        selections.length !== vimState.editor.selections.length
+          ? true
```

Comment:
nit: this ternary can be replaced with an `||`

Reply:
fixed